agent/bench-scripts/tests/pbench-fio/test-55.env

```diff
@@ -1 +1 @@
-export _BM_EXIT_STS=42
+export _BM_FIO_STS=42
```

Comment:
I'm not against this change per se, but I think it was better having a generic "interface", rather than having differently-named environment variables which do the same things for each workload.

Reply:
It seems safer to be sure the behavior request for `fio` does not impact `uperf` and vice versa.  A mistake elsewhere can lead to a leak across tests which this prevents.